src/shared/components/palette/date/write-date-container-field.component.ts

```diff
@@ -0,0 +1,8 @@
+import { AbstractFormFieldComponent } from '../base-field/abstract-form-field.component';
+import { Component, OnInit } from '@angular/core';
```

Comment:
_Nitpick:_
Same here - we don't need `OnInit` as the component doesn't implement the interface.

Reply:
Done